### DISCUSSION TITLE
How to dynamically update the accelerator on the tray menu?

### DISCUSSION THREAD

**Discussion by wkl007** [tauri-apps/tauri#6628] [Ideas]
Example

```rust
let quit = CustomMenuItem::new("quit".to_string(), "Quit").accelerator("Cmd+Q");
```

Expects to have a `set_accelerator` method like the `set_title` method

```rust
let item_handle = app.tray_handle().get_item(&id);
item_handle.set_title("Show").unwrap();

// pseudocode
item_handle.set_accelerator("Cmd+T").unwrap();
```

**Comment by FabianLars** [comment]
Hey, can you convert this to a feature request issue instead please? (No promises that this will ever be implemented, the win32 apis for example don't seem to make this easy without re-adding the whole menu)

**Reply by wkl007** [reply]
I modified this type

**Reply by FabianLars** [reply]
https://github.com/tauri-apps/tauri/issues/6643
